README.md

```diff
@@ -1,9 +1,99 @@
 
-# MewsFeed
+# MewsFeed – for the calm cats
 
-We are currently working towards the first major release of MewsFeed.
+MewsFeed is a micro-blogging platform built with [Holochain](https://holochain.org).
 
-[Collaborate on GitHub](https://github.com/orgs/GeekGene/projects/1) and [join our Discord](https://discord.gg/D3BykUZumM)!
+Visit [mewsfeed.social](https://mewsfeed.social) to be notified of our release!
+
+[Collaborate on GitHub](https://github.com/orgs/GeekGene/projects/1), [join our Discord](https://discord.gg/D3BykUZumM), and donate on [OpenCollective](https://opencollective.com/mewsfeed)!
+
+## How to use MewsFeed
```

Comment:
I feel like this content is better suited for the website -- not the github readme.